min_max.rb

```diff
@@ -0,0 +1,8 @@
+class MinMax
+  attr_reader :biggerMaxVal, :smallerMaxVal
+
+  def initialize(smallerMaxVal, biggerMaxVal)
```

Comment:
Having a class for this is not very efficient, have a look at [structs](https://ruby-doc.org/core-2.5.0/Struct.html)

Reply:
done !